include/orc/orc.hpp

```diff
@@ -22,11 +22,20 @@
 struct odrv_report {
     odrv_report(std::string_view symbol, const die* list_head);
 
-    std::string category() const;
+    std::size_t category_count() const { return _conflicting_attributes.size(); }
```

Comment:
These changes reflect the increased complexity required around ODRV category management.